README.md

```diff
@@ -71,8 +71,8 @@ docker run --link xray:xray --env AWS_XRAY_LOCATION=xray:2000 my-application
 
 ### Sampling
 Sampling rate should be a float within 0 to 1. Both 0 and 1 are acceptable.
-e.g. 0 means never sampled, 1 means always sampled, 0.3 means 30% of requests (or traces in not Rack app) will be sampled.
-The default sampling rate is undefined so you should set your own sampling rate on production systems. 
+e.g. 0 means never sampled, 1 means always sampled, 0.3 means 30% of requests (or traces in non-Rack app) will be sampled.
+e default sampling rate is undefined so you should set your own sampling rate on production systems.
```

Comment:
Are you sure for this one?: "The" -> "e"

Reply:
Ah nice catch! I'll fix it.

Reply:
Fixed in https://github.com/taiki45/aws-xray/pull/90/commits/ea4893f788f97668356b373bb03970b1e4b4f134 :bow: